src/Lumi/Components2/Image.purs

```diff
@@ -0,0 +1,98 @@
+module Lumi.Components2.Image
+  ( imageThumb
+  , small
+  , medium
+  , large
+  , extraLarge
+  , _customSize
+  , round
+  ) where
+
+import Prelude
+
+import Data.Maybe (Maybe(..), fromMaybe)
+import Data.Monoid as Monoid
+import Data.Nullable as Nullable
+import Data.String as String
+import Data.Tuple.Nested ((/\))
+import Effect.Unsafe (unsafePerformEffect)
+import Lumi.Components (LumiComponent, PropsModifier, lumiComponent, ($$$))
+import Lumi.Components.Loader (loader)
+import Lumi.Components.Spacing (Space)
+import Lumi.Components.Svg (userSvg)
+import Lumi.Components2.Box as Box
+import Lumi.Styles (style, toCSS)
+import Lumi.Styles.Box (FlexAlign(..))
+import Lumi.Styles.Box as Styles.Box
+import Lumi.Styles.Image as Styles.Image
+import Lumi.Styles.Slat hiding (_interactive,slat) as Styles.Slat
+import Lumi.Styles.Theme (LumiTheme(..), useTheme)
+import React.Basic.DOM as R
+import React.Basic.Emotion as E
+import React.Basic.Events (handler_)
+import React.Basic.Hooks (JSX)
+import React.Basic.Hooks as Hooks
+
+type ImageProps
+  = ( src :: String
+    , placeholder :: Maybe JSX
+    )
+
+imageThumb :: LumiComponent ImageProps
+imageThumb =
+  unsafePerformEffect do
+    lumiComponent "Image" { src: "", placeholder: Nothing } \props -> Hooks.do
+      theme <- useTheme
+      loaded /\ setLoaded <- Hooks.useState' false
+      pure
+        $ E.element R.div'
+            { children:
+                [ if String.null props.src
+                    -- @TODO generate a placeholder svg
+                    then fromMaybe userSvg props.placeholder
```

Comment:
Should `userSvg` be really the default? We also probably should be constraining/forcing the placeholder to fill the whole container.

Reply:
Agreed, maybe blank or grey for the default?

Reply:
Nope def not, `userSvg` is just a holder for now; per the TODO note I need to generate the "placeholder svg" (just am waiting on assets from design).

Good call re making sure it fills the container.

Reply:
Ah ok, sorry, I thought the TODO note was talking about other things 😅